webpack.config.js

```diff
@@ -13,6 +13,10 @@ Encore
     .setPublicPath('/build')
     // only needed for CDN's or sub-directory deploy
     //.setManifestKeyPrefix('build/')
+  .copyFiles({
+    from: './assets/images',
+    to: 'images/[path][name].[ext]',
```

Comment:
you must handle the hash, see https://github.com/crf-devs/resop/pull/380/files#diff-11e9f7f953edc64ba14b0cc350ae7b9dR18